### ISSUE TITLE
Can features be discarded by the classifier?

### ISSUE THREAD

**Issue by PKholyavin** [TeamHG-Memex/sklearn-crfsuite#58]
Hello!
I am using crfsuite to train models on my own datasets, and I am testing different sets of features (I have a lot of them). However, some of those features seem to have no effect on classification results: e.g. first I use set of features A and get an F1 = X, and then I use set A + B and get the same results, and this repeats on every train and test set I have (if it is any help, my data is various acoustic features of speech in two languages). My question is: is this normal, or is there a possibility that some of these features are somehow discarded by the model?
Thank you in advance!

**Comment by PKholyavin** [comment]
Update: I have checked `attributes_`, and these features are indeed not used. What could be the reason?
